src/api/routes/transcripts.ts

```diff
@@ -52,26 +53,26 @@ const getMostRecentStartedAt = procedure
   return await db.Transcript.max("startedAt", { where: { groupId } });
 });
 
-const getNameMap = procedure
+const getUserMap = procedure
```

Comment:
Because the name map is more closely associated with transcripts than summaries, I suggest we change zSumaryNameMap to zTranscriptNameMap, and also rename the SummaryNameMap model.